src/main/java/cz/cvut/kbss/termit/util/ConfigParam.java

```diff
@@ -24,6 +24,11 @@
      */
     REPOSITORY_URL("repository.url"),
 
+    /**
+     * URL of the main application repository.
```

Comment:
The comment is duplicate of the REPOSITORY_URL, which is especially unfortunate since it gives absolutely no explanation as to why there is another repository URL parameter.